src/coreComponents/dataRepository/Group.hpp

```diff
@@ -323,7 +333,10 @@ class Group
   T & getGroup( KEY const & key )
   {
     Group * const child = m_subGroups[ key ];
-    GEOSX_THROW_IF( child == nullptr, "Group " << getPath() << " doesn't have a child " << key, std::domain_error );
+    GEOSX_THROW_IF( child == nullptr,
+                    "Group " << getPath() << " has no child named " << key << std::endl <<
+                    getName() << " children are: " << dumpChildrenName(),
```

Comment:
Do you mean `"The children of " << getName() << " are: " << dumpChildrenName()`?